skrub/_gap_encoder.py

```diff
@@ -389,8 +394,12 @@ def get_feature_names_out(
         for i in range(n_components):
             x = encoding[:, i]
             labels = vocabulary[np.argsort(-x)[:n_labels]]
-            topic_labels.append(labels)
-        topic_labels = [prefix + ", ".join(label) for label in topic_labels]
+            label = ", ".join(labels)
+            # Avoid having twice the same name for the different features
+            if label in topic_labels:
```

Comment:
we store the labels in `topic_labels` with the prefix, but here we check if our label is already in there before adding its prefix. so it can still end up being a duplicate after we add the prefix, I think